index.html

```diff
@@ -45,10 +45,10 @@
       <div class="backgroundimage">
         <div class="content">
           <div class="welcometext">
-            <p>
+            <h1>
```

Comment:
Kindly change the contrast to be at least `4.5` of the `See project` buttons in `hover` because they failed in the contract check
![image](https://user-images.githubusercontent.com/24703171/156264129-66b4af60-c04d-4947-a3c7-f812752e8d4b.png)
You can check the contrast for your website using this [link](https://www.aditus.io/button-contrast-checker/)